lib/Onyx.js

```diff
@@ -1019,6 +1021,10 @@ function multiSet(data) {
  * @returns {*}
  */
 function applyMerge(existingValue, changes) {
+    if (_.some(changes, change => _.isNull(change))) {
+        return null;
+    }
```

Comment:
This seems wrong to me. If _any_ of the changes in the merge queue array are `null` then we `null` out the resulting value?

Reply:
I am not 100% sure about this one, but I _think_ it makes sense? i.e: if I'm trying to merge all these changes into an empty object:

```js
[
    {a: 1},
    {b: 2},
    null,
    {c: 3},
]
```

then what should the result be? This code makes it `null`, but maybe it should actually be applied in order such such that the final result is `{c: 3}`?

Reply:
>then what should the result be?

It should be the last value in the array

Reply:
>If null is anywhere else in the array, only the last item in the array will be applied.

Not anywhere else in the array. Specifically, it should clear out the items that came before it. The rest should merge normally.

Reply:
>If null is anywhere else in the array, it is ignored.

I think this logic is wrong. It should be clearing the previous values called in the same tick of the event loop. 

If we do this:

```js
Onyx.merge('test', {a: 1});
Onyx.merge('test', null);
Onyx.merge('test', {b: 2});
Onyx.merge('test', {c: 3});
```
Then the expected result would be `{b:2, c: 3}`?